Tools/Scripts/webkitpy/common/net/bugzilla/attachment.py

```diff
@@ -37,8 +37,8 @@
 
 class Attachment(object):
     fast_cq_preamble = "[fast-cq] "
-    revert_preamble = "REVERT of r"
-    rollout_preamble = "ROLLOUT of r"
```

Comment:
do we need to append r to svn_revisions somewhere?

Reply:
We already do in [Tools/Scripts/webkitpy/tool/commands/download.py](https://github.com/WebKit/WebKit/pull/1992/files#diff-5577606b5e5254a7108d3e3c86dcbeb56f4e53c74f2f5e0dd60245d44dd2dbd4)

```
revision_list = ['r{}'.format(rev) for rev in revisions] + commits
```

commits will contain hashes and identifiers, so doesn't need the same treatment.